recipe/vs2010_omp.patch

```diff
@@ -0,0 +1,12 @@
+--- Source/LibRawLite/libraw/libraw_types.h
++++ Source/LibRawLite/libraw/libraw_types.h
+@@ -32,7 +32,7 @@
+ #if defined (_OPENMP)
+ 
+ #if defined(WIN32)
+-# if defined (_MSC_VER) && (_MSC_VER >= 1600 || (_MSC_VER == 1500 && _MSC_FULL_VER >= 150030729) )
++# if defined (_MSC_VER) && (_MSC_VER > 1600 || (_MSC_VER == 1500 && _MSC_FULL_VER >= 150030729) )
```

Comment:
@msarahan no it was not me. There were other authors too. I think I based it on an existing recipe, which might be where its from